src/icons/rename-icon.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 15 14">
+    <defs>
+        <path id="a" d="M0 11.5h15V14H0v-2.5zM6.9.9H8l3.5 8.7h-1.4l-.8-1.8H5.5l-.7 1.8H3.4L7 1zM6 6.5h3l-1.5-4-1.5 4z"/>
```

Comment:
These ids end up being global on the page, which can sometimes cause conflicts if the names overlap. We should be careful. If possible, maybe we could rewrite this svg to inline the paths.